dradis/dradis-ce
# Evidence tab

### Spec
Currently, there is no way of deleting multiple evidences in a node in one go. 

**Proposed solution**
Adding an evidence tab in Nodes#show that has a table with checkboxes for each record should solve the problem

### How to test
1. Add a node.
2. Upload multiple evidences to the node.
3. Assert that the tab works as expected.

4. Delete multiple evidences in the node.
5. Assert that the evidences are deleted.
